src/Header/__snapshots__/Header.test.jsx.snap

```diff
@@ -32,6 +32,7 @@ exports[`Header Matches snapshot 1`] = `
           className="search"
         >
           <Search
+            onNavigating="onNavigatingHandler"
```

Comment:
Can you add a test that asserts that the handler is called when an autocomplete suggestion is confirmed? This diff asserts that the prop is being passed, but there's no test around the behaviour of the onConfirm function